### PR TITLE
Fix typos in pricing docs

### PR DESCRIPTION
**Pull request by Mansive**
## Summary

Fixes typos

### CODE REVIEW

File: 030-Concepts/080-pricing.mdx [hunk 1]
```diff
@@ -42,7 +42,7 @@ Daily backups ensure your data is regularly and automatically backed up every da
 
 #### Point-in-time recovery
 
-Point-in-time recovery (PITR) enables the restoration of data to a specific timestamp that a you designates. PITR operates by consistently backing up transaction logs, so you can revert data to any designated point. You can use PITR for the following:
+Point-in-time recovery (PITR) enables the restoration of data to a specific timestamp that you designate. PITR operates by consistently backing up transaction logs, so you can revert data to any designated point. You can use PITR for the following:
```
Comment (joan-ing) [inline 030-Concepts/080-pricing.mdx:45]: Thank you @Mansive! Missed this

File: 030-Concepts/080-pricing.mdx [hunk 2]
```diff
@@ -142,23 +142,23 @@ If you exceed the rate limit on Xata, you will receive a 429 HTTP status as a no
 
 ##### Can I increase my rate limit?
 
-Yes, you can increase the limit by upgrading to the Pro plan, if you are on the Free plan. You can also [contact us](mailto:contactus@xata.io)if you would like to discuss pricing options.
+Yes, you can increase the limit by upgrading to the Pro plan, if you are on the Free plan. You can also [contact us](mailto:contactus@xata.io) if you would like to discuss pricing options.
```
Comment (joan-ing) [inline 030-Concepts/080-pricing.mdx:145]:
```suggestion
Yes, you can increase the limit by upgrading to the Pro plan, if you are on the Free plan. You can also [contact us](mailto:contactus@xata.io) if you would like to discuss pricing options.
```

File: 030-Concepts/080-pricing.mdx [hunk 3]
```diff
@@ -171,7 +171,7 @@ A billing period lasts 30 days and begins on the day you start your plan.
 
 For Pro and Enterprise plans, an invoice will be generated every 30 days based on the date you initiated your plan. This invoice will cover the past month's usage and any previous outstanding amounts.
 
-##### Do you have a program for start-ups program?
+##### Do you have a program for start-ups?
```
Comment (joan-ing) [inline 030-Concepts/080-pricing.mdx:174]:
```suggestion
##### Do you offer a program specifically designed for start-ups?
```

**Review by joan-ing — APPROVED** [review]
LGTM! Thanks for opening this PR. It's really appreciated @Mansive

**Comment by joan-ing** [comment]
Hey @Mansive, would you mind running `pnpm install` in the terminal for this PR? It'll help set up the linter. There are instructions in the README, but if you need any help please tag me in a follow up comment. After you apply the linter we will be able to merge your PR. Thank you so much for you great contribution  and help. Best wishes for the new year and hope to see you in some of our other repos  :tada

**Comment by Mansive** [comment]
Hello @joan-ing, I'm glad to be of help. I've ran `pnpm install`, but what should I do next? I'm rather new to all this.

**Comment by joan-ing** [comment]
> Hello @joan-ing, I'm glad to be of help. I've ran `pnpm install`, but what should I do next? I'm rather new to all this.

Hi @Mansive , no worries. This isn't terribly straightforward.

From the build logs, it looks like your token might not be current. 

image
Did you previously set up a secret in GitHub? If not, you can follow these steps to create one:

To define the a Secret...

1. Go to your GitHub repository.
1. Click **Settings** in the upper right corner.
1. In the left sidebar, click **Secrets**.
1. Click **New repository secret** .
1. Enter a name for the secret (e.g., "YOUR_SECRET_NAME") and its value (the token or secret you need).
1. Click **Add secret** button to save it.

After you create the secret! I would try pushing a commit to your PR. Hopefully that will get this over the line. Thank you so much for being so patient and eager to try this

**Comment by Mansive** [comment]
@joan-ing I've created a secret called GIT_TOKEN, put my personal access token in it, and pushed an empty commit. Hopefully this is enough.

**Comment by joan-ing** [comment]
Thank you for your persistence in solving this @Mansive. Will be sure to thank you in our upcoming changelog in Discord 🥇

**Pull request merged**